lib/marc/record.rb

```diff
@@ -1,7 +1,9 @@
-module MARC  
-    
+require 'forwardable'
+
+module MARC
+
   # The FieldMap is an Array of DataFields and Controlfields.
-  # It also contains a Hash representation 
+  # It also contains a Hash representation
   # of the fields for faster lookups (under certain conditions)
   class FieldMap < Array
```

Comment:
Just wondering if this could be simplified by using http://apidock.com/rails/ActiveSupport/OrderedHash